langfuse/integrations.py

```diff
@@ -0,0 +1,83 @@
+import os
+import functools
+from datetime import datetime
+from dotenv import load_dotenv
+
+import openai
+from openai.api_resources import ChatCompletion, Completion
+
+from langfuse import Langfuse
+from langfuse.client import InitialGeneration
+from langfuse.api.resources.commons.types.llm_usage import LlmUsage
+
+
+load_dotenv()
```

Comment:
Why is this needed here? So far this worked without

Reply:
As I have a PR #100 so it will be useful there. And in `.env` we will also have OpenAI_key so load that as well.

Reply:
I left one comment there. Happy to merge that asap

Reply:
Great made changes there